src/ORM/Association/BelongsToMany.php

```diff
@@ -717,20 +717,21 @@ function () use ($sourceEntity, $targetEntities, $options) {
      * this association
      * @param bool $cleanProperty whether or not to remove all the objects in $targetEntities
      * that are stored in $sourceEntity
+     * @param array $options list of options to be passed to the internal `delete` call
      * @throws \InvalidArgumentException if non persisted entities are passed or if
      * any of them is lacking a primary key value
      * @return void
      */
-    public function unlink(EntityInterface $sourceEntity, array $targetEntities, $cleanProperty = true)
+    public function unlink(EntityInterface $sourceEntity, array $targetEntities, $cleanProperty = true, array $options = [])
```

Comment:
Are you referring to the ability to use the `$cleanProperty` argument for passing the options array? Or just to setting the `cleanProperty` key in the options array in case it doesn't already exist?